recipes/cad1/task1/baseline/enhance.py

```diff
@@ -21,16 +19,20 @@
 from clarity.utils.signal_processing import denormalize_signals, normalize_signal
 from recipes.cad1.task1.baseline.evaluate import make_song_listener_list
 
+# pylint: disable=too-many-locals
```

Comment:
The `.pylintrc` file hasn't been modified, looks like its picked up that this has been disabled in [`.pylintrc.precommit.ini`](https://github.com/claritychallenge/clarity/blob/main/.pylintrc.precommit.ini) was that automagic or did you have to do anything to configure it? There is nothing in the `.pre-commit-config.yaml` under `pyupgrade` that configures it to look at this file. :thinking: